src/options/options.html

```diff
@@ -81,8 +81,11 @@ <h6 class="margin-top-medium vocab heading">All Providers</h6>
           </div>
         </div>
         <div class="content" data-content-no="2">
-          <!-- Persist search even when extension is closed?
-          yes (checkbox) -->
+          <h6 class="vocab heading">Options</h6>
+          <div class="enable-search-storage-container">
+            <input type="checkbox" class="vocab choice-field magenta-colored small-sized" id="enable-search-storage-checkbox" />
+            <label for="enable-search-storage" class="padding-left-smaller">Enable Search Storage</label><br />
```

Comment:
Since you have change the `id` of the `input` tag, the `for` attribute of the `label` tag should also change.